src/utils/info.js

```diff
@@ -6,3 +6,10 @@ export const showToast = (message: string) => Toast.show(message, Toast.SHORT);
 
 export const showErrorAlert = (message: string, title: string) =>
   Alert.alert(title, message, [{ text: 'OK', onPress: () => {} }], { cancelable: true });
+
+export const showErrorFromException = (error: Error, title: string) =>
+  error
```

Comment:
This code looks a bit too smart for its own good.
The meaning of it is to return the value but you don't want that. `Alert.alert` has a return type of `void`.
https://github.com/facebook/react-native/blob/master/Libraries/Alert/Alert.js#L47

Related to that `showErrorAlert` is also not that correct, just doesn't look as weird :)